### DISCUSSION TITLE
Unable to locate package issue, Matlab dependencies, Ubuntu/Debian

### DISCUSSION THREAD

**Discussion by Sachin2911** [chpc-tech-eval/chpc24-scc-nationals#29] [Show and tell]
If you have come across the same set of errors while attempting to install the dependencies required for MATLAB:
![2024-12-02_00-32](https://github.com/user-attachments/assets/e4f04ecf-da53-4efb-8eab-2dac2d1794ab)
Use the following replacement command to resolve:

```bash
sudo apt install libasound2t64 libcairo2 libcairo-gobject2 libcups2 libgdk-pixbuf-2.0-0 libglib2.0-0 libc6 language-pack-en locales libgtk-3-0 libice6 libxcomposite1 libxcursor1 libxdamage1 libxfixes3 libxft2 libxinerama1 libxrandr2 libxt6 libxtst6 libxxf86vm1 libcap2 libdrm2 libgl1 libsndfile1 libltdl7 libuuid1 libwayland-client0 libgbm1 net-tools libnspr4 libnss3 libpam0g libpango-1.0-0 procps sudo unzip which zlib1g
```

The issue is that Debian renamed quite a few packages from their upstream names. @nyameko is this repo also going to be collaborative?

**Comment by nyameko** [comment]
Thanks @Sachin2911 ,

Good work on identifying and fixing this!

Yes, this repo will be collaborative. If you'd like to `FIX` this issue, please fork the repo and submit a PR from your fork, directly into `main`. Otherwise I'll fix it first thing tomorrow morning.

**Comment by Sachin2911** [comment]
Pull request submitted! Happy to help out👍️

**Comment by ShivamNaikinfosec** [comment]
I have made a discussion discussing the solution to this problem have a look at it titled Ubuntu Dependencies

You need to basically copy libraries from this git  repo https://github.com/mathworks-ref-arch/container-images/tree/main/matlab-deps then create an executable file from it and use chown and awk to resolve your issue

**Comment by ShivamNaikinfosec** [comment]
@nyameko  I have posted a discussion on this matter hope you find it useful it shows how to perform this on ubuntu linux
